### ISSUE TITLE
Mass fabricator can't be powered

### ISSUE THREAD

**Issue by axlegear** [Blood-Asp/GT5-Unofficial#296]
I have a 128v one being fed 4amps from a transformers (512v->128v) with a battery in the machine and it still runs out of power as soon as it starts.  The battery drains only briefly.

Even with 9Amps it's still draining the battery and dying out.  How many amps did you intend?

**Comment by Blood-Asp** [comment]
I guess something is wrong in my overclocking calculation. The 128v one should need 512EU + loss = 5amp.

**Comment by axlegear** [comment]
It seems to run with 13A, but very slowly drains a battery at 12A (likely due to the cable loss)

**Comment by Blood-Asp** [comment]
fixed
